corehq/apps/dump_reload/couch/load.py

```diff
@@ -54,7 +54,7 @@ def _get_db_for_doc_type(self, doc_type):
                 raise DocumentClassNotFound('No Document class with name "{}" could be found.'.format(doc_type))
             callback = LoaderCallback(self._success_counter, self.stdout)
             chunksize = 100
-            if doc_type == "Application":
+            if 'App' in doc_type:
```

Comment:
What would you think of encoding the specific list of doc types instead?

Reply:
True, that'd make it pretty clear which ones we're expecting to be large. Just making sure we're only considering `Application`, `LinkedApplication` and `RemoteApp`, right? Is there any other models we should consider that might get too large for the chunk?

Reply:
That's all of the Application doc types, I believe.  The next biggest models i can think of are CommCareUser and WebUser, but I doubt they are bad enough to cause issues like Application does.